dash/dash.py

```diff
@@ -385,11 +386,10 @@ def _relative_url_path(relative_package_path='', namespace=''):
                     self.registered_paths[resource['namespace']]\
```

Comment:
Now dynamic resources will not be registered here. I think this would break: https://github.com/plotly/dash/blob/5ba0bc25448227f33559a71adb4746ebb09bc349/dash/dash.py#L484